chatlearn/models/vllm_module.py

```diff
@@ -595,14 +600,14 @@ def data_parallel_size(self):
         """
         :meta private:
         """
-        return 1
+        return self._num_replica
```

Comment:
for each instance, the data parallel size should be 1

Reply:
done